locations/spiders/jet_gb.py

```diff
@@ -0,0 +1,9 @@
+from locations.categories import Categories
+from locations.storefinders.storerocket import StoreRocketSpider
+
+
+class JetGBSpider(StoreRocketSpider):
+    name = "jet_gb"
+    item_attributes = {"brand": "JET", "brand_wikidata": "Q568940", "extras": Categories.FUEL_STATION.value}
```

Comment:
Possible duplicate - https://github.com/search?q=repo%3Aalltheplaces%2Falltheplaces+Q568940&type=code - though this is probably the US version

Reply:
Non issue